### PR TITLE
feature/course-1

### PR DESCRIPTION
**Pull request by grojas123**
Feature/course 1 Create an Initial Website Mockup Final version .

### CODE REVIEW

**Comment by grojas123** [comment]
Created the 3 version of the site with css ubiqum-bootcamp\staff\gustavo-rojas\course-1 in nysl_[1,2,3]_css

**Comment by manuelbarzi** [comment]
hi gustavo
your three design projections seem to work fine (tested them in my local machine)
just a few comments
- you are using `id` selectors in css for everything. even though this is working, it is not the best practice to use them, but class selectors. moreover, it is a good practice that you learn a good methodology to organise the css styles in your file. pay attention to the resources shared today in our discord channel. there you will find a demo we did about BEM, which might be useful for you to do better css coding. so i encourage you to create new versions of your styles working with class selectors and BEM methodology. you can create new folders for those new versions.
- another thing to point out is the challenge of creating one single html and multiple css designs. that's the real challenge here. we shared resources about this in the discord channel too in the past days. pay attention to the CSS Zen Garden idea (this demo is The Demo of the web about this topic). the goal is: having one single html with the structure and data, provide different css styles that fit the requirements (sketches from the e-mails). so **not** multiple html files, but **just one html file**, and different css files that could be used to provide different styles to the same web.

**Comment by grojas123** [comment]
Esta es la version de la tarea 1 . La hice con CSS para ir adelatando la siguiente tarea .

File: staff/gustavo-rojas/course-1-new-version/about.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>NorCal Time Trials Series</title>
+  <link rel="stylesheet" href="styles/nctss_1.css">
```
Comment (elolivier) [inline staff/gustavo-rojas/course-1-new-version/about.html]: Please, use only tools related to Task 1, no CSS.

**Review by elolivier — CHANGES_REQUESTED** [review]

**Comment by grojas123** [comment]
Updated the \staff\gustavo-rojas\course-1-new-version\task1 with no usage of FlexBox with nctss_1.css

**Comment by grojas123** [comment]
Task2 Final Version of \staff\gustavo-rojas\course-1-new-version\task2 with usage of FlexBox with nctss_[1,2,3].css Tested done

**Comment by grojas123** [comment]
I need to fix the nctss_3.css

**Pull request closed without merging**